cscs-checks/analytics/hpcg/hpcg_benchmark.py

```diff
@@ -0,0 +1,109 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class HPCGCheckRef(rfm.RegressionTest):
+    def __init__(self, **kwargs):
+        super().__init__()
+
+        self.descr = 'HPCG reference benchmark'
+        self.valid_systems = ['daint:mc', 'daint:gpu', 'dom:gpu', 'dom:mc']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.modules = ['craype-hugepages8M']
+        self.build_system = 'Make'
+        self.build_system.options = ['arch=MPI_GCC_OMP']
+        self.sourcesdir = None
+        self.prebuild_cmd = ['git clone https://github.com/hpcg-benchmark/hpcg.git', 'cd hpcg']
```

Comment:
Better use the `sourcesdir` alternative to clone the repository, as explained in https://eth-cscs.github.io/reframe/advanced.html?highlight=git#retrieving-the-source-code-from-a-git-repository

Reply:
ok